chapters/remote/jinad.md

```diff
@@ -1,88 +1,75 @@
 
-## Terminologies
-
-- *Workflow*: a set of connected pods for accomplishing certain task, e.g. indexing, searching, extracting.
-- *Flow API*: a pythonic way for users to construct workflows in Jina with clean, readable idioms. 
-- *Remote*, *remote instance*, *remote machine*: the place where you want to run the pod, the place offers better computational capability or larger storage. For example, one may want to run an encode pod on the remote GPU instance.  
-- *Local*, *local instance*, *local machine*: the place of your entrypoint and the rest parts of your workflow.
-
-## Design
-
-`jinad` is designed to maintain bookkeeping for the running Flows, Pods and Peas on the remote machines. `jinad` can also spawn Pods and Peas to other remote machines that have `jinad` running. [`fluentd`](https://github.com/fluent/fluentd) is used to collect logs from different Processes and ensure the logs belonging to the same Flow are stored consistently. 
-
-![jinad design](jinad_design.png)
+![JinaD design](jinad_design.png)
 
 ## Installation
 
-### Using Docker Images (Recommended)
+##### Docker Image (Recommended)
 
-The simplest way to use `jinad` is via Docker. There is no need to worry about [`fluentd`](https://github.com/fluent/fluentd) installation. You only need to have [Docker installed](https://docs.docker.com/install/) first. 
+The simplest way to use `JinaD` is via Docker. You only need to have [Docker installed](https://docs.docker.com/install/).
 
-In the command below, we use the tag `latest-daemon` which uses the latest release version of `jinad`. Of course, you can switch to other versions, and you can find all the available versions at [hub.docker.com](https://hub.docker.com/repository/docker/jinaai/jina/tags?page=1&ordering=last_updated&name=daemon). You can found more information about the versioning at [github.com/jina-ai/jina](https://github.com/jina-ai/jina/blob/master/RELEASE.md).
+We use the tag `latest-daemon` in the below command, which uses the latest release of `JinaD`. To use other versions, please refer to [versioning in Jina](https://github.com/jina-ai/jina/blob/master/RELEASE.md) and find all the available versions at [Docker Hub](https://hub.docker.com/repository/docker/jinaai/jina/tags?page=1&ordering=last_updated&name=daemon).
 
 ```bash
 docker pull jinaai/jina:latest-daemon
 ```
 
-### Using PyPi package
+##### PyPi package
 
-> Notes: As one part of the jina package, `jinad` follows the same [installation instructions of jina](https://docs.jina.ai/chapters/install/os/via-pip.html) and you only need to cherry pick `[daemon]`
+> Note: As one part of the jina package, `JinaD` follows the same [installation instructions of jina](https://docs.jina.ai/chapters/install/os/via-pip.html) and you only need to cherry pick `[daemon]`
```

Comment:
'JinaD' is a part of the jina package and it follows the same installation instructions as jina.'

Not sure what cherry pick refers to here (like git cherry pick?) so id restructure the second part of the sentence to be more clear

Reply:
Changed to "pick". Command to install jinad is mentioned below.